justfile

```diff
@@ -6,6 +6,13 @@ test:
 	# Run the tests.
 	GODEBUG=cgocheck=2 go test -test.v
 
+examples:
+	#!/usr/bin/env bash
+	#export DYLD_PRINT_LIBRARIES=y
+	cd examples
+	# Run the examples.
+	GODEBUG=cgocheck=2 go test
```

Comment:
This run all the tests, not only the examples, right?

Reply:
only the examples as we are in the `examples/` directory.